test/bin/invalid-package-version.t/run.t

```diff
@@ -41,7 +41,7 @@ to the test)
   $ opam-monorepo lock toonew 2> errors
   ==> Using 1 locally scanned package as the target.
   [1]
-  $ grep -Pazo "(?s)opam-monorepo: \[ERROR\].*(?=opam-monorepo)" < errors | head --bytes=-1
+  $ perl -0ne 'print $1 if /(?s)(opam-monorepo: \[ERROR\].*)opam-monorepo/s' < errors | head --bytes=-1
```

Comment:
Probably would be nicer to have a separate program process this, since it's becoming increasingly difficult to understand what is being processed here (and I say that having written the initial line which, a few month down the line looks also puzzling)

Reply:
I simplified the regex so it matches without the perl regexp extension to grep (the `-P`).